automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/SignInSpec.scala

```diff
@@ -11,24 +11,16 @@ class SignInSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Mat
 
   implicit override val patienceConfig = PatienceConfig(timeout = scaled(Span(5, Seconds)), interval = scaled(Span(500, Millis)))
 
-  "A user" - {
-    "with a registered account" - {
+  "A user with a registered account" - {
 
-      "should be able to log in and out multiple times as multiple users" in {
-        val Seq(user1, user2) = UserPool.chooseStudents(2)
+      "should be able to log in and out" in {
+        // "harry": "harry.potter@quality.firecloud.org"
```

Comment:
restrict Google real SignIn to use just one user. It's painful to use x11 forwarding for Google SignIn security question.

Reply:
We need 2 users, otherwise we lose coverage of the behavior of making sure sign-out followed by sign-in works correctly. The chance that this will fail is low but the cost if it did could be very high!